### PR TITLE
Feature/45421 - Galaxy adapter

### PR DESCRIPTION
**Pull request by mjk90**
## Description
Added Galaxy WS adapter & Galaxy adapter tests

## Steps to Test

1. `yarn test packages/sources/galaxy/test`

## Quality Assurance

- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [x] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [x] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)
- [x] All code changes have 100% unit and integration test coverage. If testing is not applicable or too difficult to justify doing, the reasoning should be documented explicitly in the PR.

### CODE REVIEW

File: packages/sources/galaxy/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
+# @chainlink/example-source-adapter
```
Comment (justinkaseman) [inline packages/sources/galaxy/CHANGELOG.md:1]:
```suggestion
# @chainlink/galaxy-adapter
```
& clear file of updates. Automation on this coming this week.

File: packages/sources/galaxy/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@chainlink/galaxy-adapter",
+  "version": "0.2.10",
```
Comment (justinkaseman) [inline packages/sources/galaxy/package.json:3]:
```suggestion
  "version": "0.0.0",
```
Automation on this coming this week.

File: packages/sources/galaxy/src/adapter.ts [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        if (!subscriptionMessage.signals.includes(message.signal)) return
+        return { type: 'subscribe', signals: subscriptionMessage.signals }
+      },
+      isError: (message: TickerMessage) => message.type === 'error',
+      filter: (message: TickerMessage) => message.type === 'signal_update',
+      toResponse: (message: TickerMessage, input: AdapterRequest) => {
+        const result = Requester.validateResultNumber(message, ['value'])
+        return Requester.success(input.id, { data: { result, ts: message.ts } }, config.verbose)
+      },
+    } as any // TODO: connection type mismatch
```
Comment (austinborn) [inline packages/sources/galaxy/src/adapter.ts:107]: Connection type mismatch?

**Review by austinborn — APPROVED** [review]
LGTM!

File: packages/sources/galaxy/src/adapter.ts (hunk 3 above)
Comment (austinborn) [reply packages/sources/galaxy/src/adapter.ts:107]: Oh I see, it's the same issue as with GSR. I think these can be handled as a unit in a separate PR update.

**Pull request merged**